src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/PublishContent.targets

```diff
@@ -10,13 +10,18 @@
   <!-- gathers the items to be published -->
   <Target Name="GatherItemsForPattern">
     <Error Condition="'$(PublishPattern)' == ''" Text="Please specify a value for PublishPattern using standard msbuild 'include' syntax." />
+
+    <PropertyGroup>
+      <ReleativeBlobPathBase>$(BlobNamePrefix)</ReleativeBlobPathBase>
+      <ReleativeBlobPathBase Condition="'$(ReleativeBlobPathBase)' != '' and !HasTrailingSlash('$(ReleativeBlobPathBase)')">$(ReleativeBlobPathBase)</ReleativeBlobPathBase>
```

Comment:
Should there be a slash after the variable here? `...">$(ReleativeBlobPathBase)</...`